docs/source/usage.rst

```diff
@@ -142,6 +142,13 @@ below the right window.
 
 ..  figure:: ./images/use/image.png
 
+Citation
+~~~~~~~~
```

Comment:
Thanks for working on this! I think there is no need for "Citation" headline. Think of the citation as a kind of thank-you note to the group of people publishing the data, and even more so a professional acknowledgement. It should be clear from this short note how we are using the data.

For actually referencing the data set, you can and should use the preconfigured citation system - see the [holography docs](https://libertem.github.io/LiberTEM/app/holography.html#hologram-simulation) for an example. For that to work, you need to add [the bibtex entry](https://zenodo.org/record/3592520/export/hx) of the data set to our bib file. This will also make the data set appear, properly formatted and liked, on [the list of citations](https://libertem.github.io/LiberTEM/index.html#citations) in our documentation, and later in the list of citations [on our zenodo entry](https://zenodo.org/record/3674003)

For another example, see also the beginning of https://nbviewer.jupyter.org/github/LiberTEM/LiberTEM-blobfinder/blob/master/examples/strainmap-poly.ipynb - although this is not using the bibtex citing system, as that is not so easily possible from a notebook.

/cc @uellue

Reply:
Thx! I have never used citation, Now it's very clear. Updated PR with requested changes.